.circleci/config.yml

```diff
@@ -303,52 +311,57 @@ workflows:
       - test_dapp_unit:
           requires:
             - build_dapp
-      - deploy_gh_pages:
-          requires:
-            - test_dapp_unit
-          context: "Raiden-LC-Deploy"
+
+  publish_staging:
+    jobs:
+      - checkout:
           filters:
             branches:
               only:
                 - master
+      - install:
+          requires:
+            - checkout
+      - build_sdk:
+          requires:
+            - install
+      - build_dapp:
+          mode: "ghpages"
+          requires:
+            - install
+      - deploy_gh_pages:
+          deploy_base: "staging"
+          requires:
+            - build_dapp
 
-  publish:
+  publish_release:
     jobs:
       - checkout:
           filters:
-            <<: *filter-version-regex
+            tags:
+              only: /^v\d+\.\d+\.\d+$/
+            branches:
+              ignore: /.*/
```

Comment:
You sure `branches` & `tags` conditions is ORed? We need to try a release to check if this works.

Reply:
Well this is the old filter I just copy-pasted. So I haven't changed the behavior in that detail. But after your comment I was reading the docs again in more detail. I fail to find an exact statement. Just the proposal that in future they should act as AND.

Anyway you pointed me to the fact that for some reason I did not remove the original anchor/template (even I thought I had).